docs/Examples/bpmn-examples/bpmn-properties.md

```diff
@@ -10,3 +10,7 @@ grand_parent: "Examples"
 ---
 
 # BPMN Properties
```

Comment:
The example should contain the full example (README of the repository) _and_ additionally link to the code. Otherwise having a full page per example is a waste.

Reply:
Done 👍 I updated the Theming and Bpmn Properties examples. It makes sense since more text we have -> more visible in search

Reply:
So we duplicate the `README.md` of the example. I don't really like that. 
Is there not maybe a jekyll feature to embed / just update to a given uri (which would be the `README.md`)?

Reply:
This is a good point Max. I found a Jekyll plugin for this but we still need to spend some time to investigate wheter it is possible to use this plugin with the Github pages + Just-The-Docs theme. I opened this ticket about this topic: https://github.com/bpmn-io/docs.bpmn.io/issues/27